### PR TITLE
Homepage update

### LINKED ISSUE #24: PPT-011 - Update homepage to include brand information, guides, about us, etc.
Using tailwind template and integrating it with our code

### PR DESCRIPTION
**Pull request by gaoqiangzhou**
- added homepage intro
- added homepage features
- added homepage footers 
- updated navbar

### CODE REVIEW

**Comment by aappacode** [comment]
@gaoqiangzhou to add transparent background to navBar (remove any color but ensure padding remains)

**Comment by gaoqiangzhou** [comment]
> @gaoqiangzhou to add transparent background to navBar (remove any color but ensure padding remains)

I think you mean change the navBar color to the current page color, so the color is consistent?
if that is the case, it is fixed

**Comment by aappacode** [comment]
> > @gaoqiangzhou to add transparent background to navBar (remove any color but ensure padding remains)
> 
> I think you mean change the navBar color to the current page color, so the color is consistent? if that is the case, it is fixed

Okay, I see the code changes, great!!

**Comment by aappacode** [comment]
@gaoqiangzhou I updated some colors and text, is this okay? (For the footer, I just changed the hover/text color)
Screenshot 2023-11-08 at 8 55 24 PM
Screenshot 2023-11-08 at 8 55 49 PM
Screenshot 2023-11-08 at 8 55 59 PM

File: src/components/HomeGuide.jsx
Comment (aappacode) [inline src/components/HomeGuide.jsx:1]: good use of components and listing tabs

File: src/components/NavBar.jsx [hunk 1]
```diff
@@ -14,42 +118,37 @@ const NavBar = () => {
...
-              onClick={handleLogout}
-              className="block py-2 pl-3 pr-4 text-gray-900 rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0 dark:text-white md:dark:hover:text-blue-500 dark:hover:bg-gray-700 dark:hover:text-white md:dark:hover:bg-transparent">
-              Log Out
-            </button>
-            }
-          </ul>
-        </div>
+  useEffect(() => {
+    if(location.pathname == '/') setBgColor(" bg-[#D3F6DB]")
+    else setBgColor(" bg-transparent")
```
Comment (aappacode) [inline src/components/NavBar.jsx:123]: this code works but we will have to check it against the sign in/sign up page Samira made, but this is okay for now

File: src/components/HomeIntro.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    const handleLogout = async () => {
+        try {
+            await logout();
+            navigate("/");
+        } catch (err) {
+            console.error("Error logging out:", err);
+        }
+    };
+
+    const colors = {
```
Comment (aappacode) [inline src/components/HomeIntro.jsx:18]: is there a reason for including these colors here?

**Review by aappacode — APPROVED** [review]
Left a few comments for when we do manual E2E testing/checking if some code parts are needed!

**Pull request merged**